npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -5,7 +5,7 @@ module NpmAndYarn
     module Helpers
       def self.npm_version(lockfile_content)
         return "npm7" unless lockfile_content
-        return "npm7" if JSON.parse(lockfile_content)["lockfileVersion"] == 2
+        return "npm7" if JSON.parse(lockfile_content)["lockfileVersion"] >= 2
```

Comment:
Do we reference this in places?
```suggestion
        return "npm8" if JSON.parse(lockfile_content)["lockfileVersion"] >= 2
```

Was also wondering if it would make sense to ship an incremental fix with this changed to `>= 2` but keep npm7 installed, if it's backwards compatible?

Reply:
Yeah that was my thinking. Npm 7 should already support the v3 format

Reply:
I had a go at downgrading to npm7 with your tests, looks 🍏  https://github.com/dependabot/dependabot-core/pull/4630 - thougts on shipping this first then figuring out the test failures for npm8?

Reply:
Yeah I think that's a great call, let's do it!